Frends.MicrosoftSQL.ExecuteProcedure/Frends.MicrosoftSQL.ExecuteProcedure/Definitions/Input.cs

```diff
@@ -14,6 +14,7 @@ public class Input
     /// </summary>
```

Comment:
https://github.com/FrendsPlatform/Frends.MicrosoftSQL/blob/a4c607221cb26c9dd36a2dc7c9db54caa36a104c/Frends.MicrosoftSQL.ExecuteProcedure/Frends.MicrosoftSQL.ExecuteProcedure/Definitions/Input.cs#L3
This namespace is not used, so it can be removed.